src/main/resources/logback.xml

```diff
@@ -42,6 +42,12 @@
         <appender-ref ref="SYSLOG"/>
     </logger>
 
+    <logger name="org.broadinstitute.dsde.rawls.dataaccess.HttpGooglePubSubDAO" level="debug" additivity="false">
+        <appender-ref ref="file"/>
+        <appender-ref ref="console"/>
+        <appender-ref ref="SYSLOG"/>
+    </logger>
+
```

Comment:
I suggest info level for the default, but adding a comment above that says "setting this to debug will log every message" which would definitely be useful to know.